### PR TITLE
Add x9 service bit support to dnsseed.bluematt.me

### PR DESCRIPTION
**Pull request by TheBlueMatt**

### CODE REVIEW

**Comment by MarcoFalke** [comment]
ACK e5f8b0d

File: src/chainparams.cpp [hunk 1]
```diff
@@ -113,12 +113,13 @@ class CMainParams : public CChainParams {
         assert(consensus.hashGenesisBlock == uint256S("0x000000000019d6689c085ae165831e934ff763ae46a2a6c172b3f1b60a8ce26f"));
         assert(genesis.hashMerkleRoot == uint256S("0x4a5e1e4baab89f3a32518a88c31bc87f618f76673e2cc77ab2127b7afdeda33b"));
 
-        vSeeds.push_back(CDNSSeedData("bitcoin.sipa.be", "seed.bitcoin.sipa.be", true)); // Pieter Wuille
-        vSeeds.push_back(CDNSSeedData("bluematt.me", "dnsseed.bluematt.me")); // Matt Corallo
+        // Note that of those with the service bits flag, most only support a subset of possible options
+        vSeeds.push_back(CDNSSeedData("bitcoin.sipa.be", "seed.bitcoin.sipa.be", true)); // Pieter Wuille, only supports x1, x5, x9, and x13
```
Comment (MarcoFalke) [inline src/chainparams.cpp]: Are you sure @sipa supports x13. (Right now I don't get an answer)

**Comment by laanwj** [comment]
Can we merge "Marking filter support for cdecker's DNS seed #8941" into this? Would make sense to do this update at the same time.

**Comment by TheBlueMatt** [comment]
Included #8941.

**Comment by cdecker** [comment]
Thanks Matt, closing #8941

**Comment by laanwj** [comment]
utACK 2449e12

**Comment by MarcoFalke** [comment]
utACK 2449e12

**Comment by btcdrak** [comment]
utACK 2449e12681ab549695f143d9b3ad2c24be03a78d

**Comment by sipa** [comment]
utACK

**Comment by jonasschnelli** [comment]
post merge ACK

**Pull request merged**